tests/basics/bytes_count.py

```diff
@@ -52,3 +52,5 @@ def t():
     return True
 
 print(b"0000".count(b'0', t()))
+
+print(bytearray(b"aaaa").count(b"a"))
```

Comment:
Newlines at the end of all four files (too many on bytearray_byte_operations.py, missing on the other three).